platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/app/OtterExecutionLayer.java

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+package org.hiero.otter.fixtures.app;
+
+import com.hedera.hapi.platform.event.StateSignatureTransaction;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.swirlds.metrics.api.Metrics;
+import com.swirlds.platform.builder.ExecutionLayer;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.List;
+import org.hiero.consensus.model.status.PlatformStatus;
+import org.hiero.consensus.transaction.TransactionPoolNexus;
+import org.hiero.otter.fixtures.TransactionFactory;
+
+/**
+ * An implementation of the {@link ExecutionLayer} for the Otter tests.
+ */
+public class OtterExecutionLayer implements ExecutionLayer {
+    /** The maximum number of transaction to store in the transaction pool */
+    private static final int TX_QUEUE_SIZE = 100_000;
+
+    /** the transaction pool, stores transactions that should be sumbitted to the network */
+    private final TransactionPoolNexus transactionPool;
```

Comment:
We can keep this for now, but I think long-term, we do not want the `TransactionPoolNexus` in Otter.